doc/source/contributing.rst

```diff
@@ -126,7 +126,7 @@ Some hints on how to run the integration tests:
 1. On Ubuntu it's probably easiest to use the `Multipass`_ snap.
 2. Launch an LTS instance using ``multipass launch -n foo``
 3. Shell into the instance: ``multipass exec foo -- bash``
-4. Install tox and python2.7: ``sudo apt install python-tox python-2.7``
+4. Install tox and python3.6+: ``sudo apt install python-tox python3-dev``
```

Comment:
I think the package is just `tox` now, but not sure from which ubuntu release.

Reply:
I'll just include the instruction and leave folks to find the install docs if they need it.